assets/src/scss/core-overrides/_heading.scss

```diff
@@ -0,0 +1,19 @@
+.wp-block-heading {
+  &.is-style-chevron::after {
+    content: "";
+    pointer-events: none;
+    margin-inline-start: $sp-1;
+    height: 12px;
+    width: 12px;
+    display: inline-block;
+    mask-image: url("../../images/chevron.svg");
+    mask-repeat: no-repeat;
+    mask-size: contain;
+    background-repeat: no-repeat;
+    background-color: currentColor;
+
```

Comment:
Remove `vertical-align: middle;` from [this line](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/core-overrides/Heading.scss#L13), since it does not take any effect.